TT Lab/Rendering/EditingContext.cs

```diff
@@ -0,0 +1,336 @@
+﻿using GlmSharp;
+using MS.WindowsAPICodePack.Internal;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
```

Comment:
Clean up the usings using Visual Studio to get rid of unused namespaces
![image](https://github.com/NeoKesha/TT-Lab/assets/23398959/82607b05-5024-41dd-872a-c95189defee8)